gslib/tests/test_cp.py

```diff
@@ -1744,10 +1744,6 @@ def _ManifestUpload(self, file_name, object_name, manifest_name):
     self.assertEqual(expected_headers, lines[0].strip().split(','))
     results = lines[1].strip().split(',')
 
-    for header in results:
```

Comment:
This did nothing, so I'm up for taking it out.  Changing the copy of the thing you're iterating over (the `header` variable, here) doesn't actually change the original thing.

```
>>> l = [1, 2, 3]
>>> for item in l:
...   l = 9
...
>>> l
[1, 2, 3]
```

So, that being said, I don't think this will actually catch any new problems.

Reply:
Good catch! I'll poke around a little more and figure out how the `b'value'` thing slipped by this integ test's asserts

Reply:
Oh whoops, I see that the assert is using a substring of the field: https://github.com/GoogleCloudPlatform/gsutil/blob/e1f413d43b1e7f4f119949b29f1acb2303c37123/gslib/tests/test_cp.py#L1749

I'll make it use the full field so it would include any weird artifacts. :)

Reply:
Ok, now it's asserting that the full source and dest buckets match!